app/src/main/java/org/eyeseetea/malariacare/SplashScreenActivity.java

```diff
@@ -55,34 +58,25 @@ private void init() {
         SdkQueries.createDBIndexes();
         PostMigration.launchPostMigration();
 
-        //Get maximum total of questions
-        if (!Tab.isEmpty()) {
-            Session.setMaxTotalQuestions(Program.getMaxTotalQuestions());
-        }
-
         if (!BuildConfig.multiuser) {
-            Log.i(TAG, "Creating demo login from dashboard ...");
-            IAuthenticationManager authenticationManager = new AuthenticationManager(this);
-            LoginUseCase loginUseCase = new LoginUseCase(authenticationManager);
-
-            Credentials demoCrededentials = Credentials.createDemoCredentials();
+            Log.i(TAG, "Pull on SplashScreen ...");
 
-            loginUseCase.execute(demoCrededentials, new ALoginUseCase.Callback() {
+            PullController pullController = new PullController(getApplication().getApplicationContext());
+            PullUseCase pullUseCase = new PullUseCase(pullController);
+            pullUseCase.execute(false, new PullUseCase.Callback() {
```

Comment:
@idelcano from here, execute method of pull should be with parameter isDemo equal true because if not execute pull from server.